polkadot-protocol/architecture/system-chains/bridge-hub.md

```diff
@@ -1,4 +1,47 @@
 ---
 title: Bridge Hub
-description: TODO
----
\ No newline at end of file
+description: Learn about the Bridge Hub system parachain, a parachain that facilitates the interactions from Polkadot to the rest of Web3.
+---
+
+## Introduction
+
+The Bridge Hub system chain is responsible for providing the necessary functionality for the relay chain (and parachains) 
+to enable trustless bridging between chains like Polkadot, Kusama, and Ethereum. It includes a set of [pallets](../../glossary.md#pallet) which facilitate the sending and receiving of messages, implementations of BEEFY, GRANDPA, and Ethereum light clients to follow and verify the state of both the source and target chain. BridgeHub also provides XCM support for target and source chains, provided XCM is implemented on the target chain.
+
+## Trustless Bridging
+
+BridgeHub provides a mode of trustless bridging through its implementation of on-chain light clients and trustless relayers. The target chain and source chain both provide ways of verifying the state of one another, of which actions (such as a transfer) are based upon the consensus and finality of both chains rather than an external mechanism controlled by a third party.
```

Comment:
We should stick to either "BridgeHub" or "Bridge Hub". IMO, "Bridge Hub" would be better since we also have "Asset Hub" in the same format